src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -342,6 +342,10 @@ akka {
 	  # this is designed to make it easy to support private / public addressing schemes
 	  public-hostname = ""
 
+	  # If set to true, we will use IPV6 addresses upon DNS resolution for host names.
+	  # Otherwise, we will use IPV4.
+	  dns-use-ipv6 = false
```

Comment:
New setting added to Remote.config - sets the default resolution strategy to IPV4, which restores backwards compatibility with previous versions of Akka.NET.